homework19/hw19_task5.py

```diff
@@ -0,0 +1,51 @@
+# <HW19> Task5. Create classes from Task 1 using type() function.
+def constructor(self, name):
+    self.name = name
+
+
+TheBot = type(
+    "Bot",
+    (object,),
+    {
+        "__init__": constructor,
```

Comment:
nit: could be implemented using `lambda`
```suggestion
        "__init__": lambda self, name: setattr(self, "name", name),
```

Reply:
Thank you!